src/commands/create-proxy.js

```diff
@@ -1,28 +1,37 @@
+'use strict';
+
 import createProxy from '../scripts/create-proxy'
 import runWithTruffle from '../utils/runWithTruffle'
 import { parseArgs } from '../utils/input'
 
-module.exports = function(program) {
-  program
-    .command('create <alias>')
-    .usage('<alias> --network <network> [options]')
-    .description(`Creates a new proxy for the specified implementation.
-      Provide the <alias> name you used to register your contract.`)
-    .option('--init [function]', "Tell whether your contract has to be initialized or not. You can provide name of the initialization function. If none is given, 'initialize' will be considered by default")
-    .option('--args <arg1, arg2, ...>', 'Provide initialization arguments for your contract if required')
-    .option('-f, --from <from>', 'Set the transactions sender')
-    .option('-n, --network <network>', 'Provide a network to be used')
-    .option('--force', 'Force creation of the proxy even if contracts have local modifications')
-    .action(function (contractAlias, options) {
-      let initMethod = options.init
-      if(typeof initMethod === 'boolean') initMethod = 'initialize'
+const signature = 'create <alias>'
+const description = 'deploys a new upgradeable contract instance. Provide the <alias> you added your contract with'
+
+module.exports = {
+  signature, description,
+  setup: function(program) {
+    program
+      .command(signature, {noHelp: true})
+      .usage('<alias> --network <network> [options]')
+      .description(description)
+      .option('--init [function]', `call function after creating contract. If no name is given, 'initialize' will be used`)
+      .option('--args <arg1, arg2, ...>', 'provide initialization arguments for your contract if required')
```

Comment:
Should we add clarifications on the format? Right now they are parsed as JSON, following what remix does. Maybe add an example even?

Reply:
alejo is going to add an example in the docs BTW

Reply:
+1, let's keep the cli help shorter, and expand on docs